tools/aqrdev

```diff
@@ -348,17 +348,18 @@ def cmd_start(ctx: AppCtx, name: str, conservative: bool) -> None:
 
 
 @app.command("stop")
+@click.option("-f", "--force", flag_value=True)
```

Comment:
Just wondering why 'force'? Maybe '--interactive' with defaults True, is better?

Reply:
Alternatively, '-n', '--non-interactive'.

Reply:
Sorry for the late response. Just thought ``--force`` is more common. I really didn't thought about interactive as in my head this sounds like something where someone can really be interactive with like a REPL and not just choose between yes or no which will result in execution or not.

Nevertheless I'm open to ideas and can change it to ``--interactive`` if this works better for most devs.

To keep it simple leave a thumbs up on my comment or a thumbs up on kyr's comment above. Thanks in advance :)

Reply:
Changed the `interactive` variable in all used methods to the counterpart `force` in order to have consistency in the code base.